packages/pill/src/component.tsx

```diff
@@ -1,67 +1,41 @@
-import React from 'react';
-import { classNames } from '@chbphone55/classnames';
-import { PillProps } from '.';
-import { pill as ccPill } from '@warp-ds/css/component-classes';
-import { i18n } from '@lingui/core';
-import { messages as nbMessages} from './locales/nb/messages.mjs';
-import { messages as enMessages} from './locales/en/messages.mjs';
-import { messages as fiMessages} from './locales/fi/messages.mjs';
-import { activateI18n } from '../../i18n';
+import React from 'react'
+import { classNames } from '@chbphone55/classnames'
+import { PillProps } from '.'
+import { pill as ccPill } from '@warp-ds/css/component-classes'
 
 export function Pill(props: PillProps) {
-  activateI18n(enMessages, nbMessages, fiMessages);
   return (
-    <div className={ccPill.pill}>
+    <div
+      role='img'
+      aria-label={props['aria-label'] === '' ? undefined : props['aria-label']}
```

Comment:
Now that I think about it, `role="img"` should always be combined with an attribute that describes such "image". Here we'd use `aria-label` but we know it can be `undefined`. I think this component should only accept `role` as props, in case you want to describe contents of the Pill with custom `aria-label`. 

Based on the [Jira ticket](https://nmp-jira.atlassian.net/browse/WARP-332) I see the accessibility team suggested that we use `role=img` and `aria-label` instead of `title`, which I think referred only to the story `PillsWithIcon`, and not to all Pill instances.

Reply:
I think we still need to have `aria-label` as a prop on pill otherwise it will not understand what `aria-label` is when using it on the `<Pill>` tag.

Reply:
Sounds good!